packages/frontend/rollup.config.mjs

```diff
@@ -12,14 +12,11 @@ import replace from "@rollup/plugin-replace";
 
 config();
 
-const reactAppEnvs = Object.entries(process.env).reduce(
-    (envs, [name, value]) => {
-        if (name.startsWith("REACT_APP"))
-            envs[`process.env.${name}`] = JSON.stringify(value);
-        return envs;
-    },
-    {}
-);
+const getEnv = (name, required) => {
```

Comment:
Should we move this function to a separate file? Since it's also used in `craco.config.js` file?